src/holo-test

```diff
@@ -39,9 +39,9 @@ run_testcase() {
     # setup chroot for holo run
     rm -rf -- target/
     cp -R source/ target/
+    mkdir -p target/run
+    mkdir -p target/tmp
     mkdir -p target/usr/share/holo/files
-    mkdir -p target/usr/share/holo/run-scripts
-    mkdir -p target/usr/share/holo/users-groups
```

Comment:
Okay, I can see that. But I still find it ugly to have to use `env.sh` or `.keep` files in every unit test for the other plugins. (Of course, that issue always existed for any non-official plugins.) What about the following replacement?

```bash
# create resource directories for all plugins in this test's holorc
grep -o 'plugin [a-z0-9-]\+' test/01-basic/target/etc/holorc | while read _ PLUGIN; do
  mkdir -p target/usr/share/holo/$PLUGIN
done
```